src/settings/Settings.js

```diff
@@ -118,6 +118,12 @@ export const SETTINGS = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "feature_message_editing": {
```

Comment:
Don't forget a PR to add a section to LABS.md in riot-web for this

Reply:
Added it in https://github.com/vector-im/riot-web/pull/9728